### PR TITLE
[animations] Adds opaque argument to OpenContainer

### PR DESCRIPTION
**Pull request by HelloJunWei**
Adds opaque argument to ``OpenContainer``, by default opaque  is true.

Please take a look.

### CODE REVIEW

**Comment by HelloJunWei** [comment]
@googlebot I fixed it.

**Comment by HelloJunWei** [comment]
@googlebot I fixed it.

File: packages/animations/lib/src/open_container.dart [hunk 1]
```diff
@@ -246,6 +248,9 @@ class OpenContainer<T extends Object> extends StatefulWidget {
   /// to the nearest navigator.
   final bool useRootNavigator;
 
+  /// By default, [opaque] is true
```
Comment (goderbauer) [inline packages/animations/lib/src/open_container.dart:251]: This needs to document what opaque does.

**Comment by goderbauer** [comment]
What's the use case for setting opaque to false for the OpenContainer?

**Comment by HelloJunWei** [comment]
Hi @goderbauer, 

Because I try to copy App Store transition using Flutter. My first version was using ``Hero`` and ``AnimatedBuilder``, However I have to control the ``text seciton`` animation to make it smoothly.
Here is the result.





I think maybe I can use ``OpenContainer`` to copy App Store transition, because the animation is more smoothly than mine.

When I scroll down the page, the ``Container`` will be small and close the ``Container``. At this situation, I want to see the page behind the ``Container`` just like the App Store. Then I need to set ``opaque``  to false in this case.

**Comment by goderbauer** [comment]
I see. Can you address the review comments?

File: packages/animations/lib/src/open_container.dart (hunk 1 above)
Comment (HelloJunWei) [reply packages/animations/lib/src/open_container.dart]: Done, please have a look.

**Comment by HelloJunWei** [comment]
Done, please have a look.

**Comment by goderbauer** [comment]
Looks like the analyzer is unhappy. This may need to be rebased to the latest master to fix this.

**Comment by HelloJunWei** [comment]
> Looks like the analyzer is unhappy. This may need to be rebased to the latest master to fix this.
oh~ I will fix it.

**Comment by goderbauer** [comment]
Looks like something went wrong with this PR, maybe a bad merge? It has 123 changed files now with many seemingly unrelated changes?

**Comment by HelloJunWei** [comment]
I just follow [CONTRIBUTING](https://github.com/flutter/packages/blob/master/CONTRIBUTING.md#contributing-code) and step by step to Contribute code.

First. ```git fetch upstream```.
Second ```git checkout upstream/master -b <name_of_your_branch>```. 
Last I make a PR, so that fixed my CI/CD issue.

Do I do something wrong?

**Comment by goderbauer** [comment]
Not sure what went wrong. Maybe open a new PR with just your changes?

**Pull request closed without merging**